database/benchmarking/benchmarking_suite/src/main/scala/uk/ac/ebi/eva/BenchmarkingSuite/MongoDBConnectionParams.scala

```diff
@@ -0,0 +1,16 @@
+package uk.ac.ebi.eva.BenchmarkingSuite
```

Comment:
I would recommend splitting the code in 2 subpackages `cassandra` and `mongodb` because some classes have a very delimited scope, and that would make the code structure more clear specially when we add 2 more DBMS to the mix.